easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart

```diff
@@ -217,8 +214,7 @@ class NFTsListTile extends StatelessWidget {
                     },
                   ),
                 ),
-              )
-            : const IgnorePointer(
+              ) else const IgnorePointer(
```

Comment:
widget should start from new line.

Reply:
No necessary. Thats the thing that formatter decides.